src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -293,56 +305,64 @@ private void incomingPacketInfo(
         // Check if incoming bitrate estimate is valid, and if it
         // needs to be reset.
         long incomingBitrate_ =
-                incomingBitrate.getRate(arrivalTimeMs);
+            incomingBitrate.getRate(arrivalTimeMs);
+
+
         if (incomingBitrate_ != 0)
         {
             incomingBitrateInitialized = true;
-        } else if (incomingBitrateInitialized)
+        }
+        else if (incomingBitrateInitialized)
         {
             // Incoming bitrate had a previous valid value, but now not
             // enough data point are left within the current window.
```

Comment:
where is the logic that tests that there is no longer enough data points left in the current window? is there a test for that somewhere?  or maybe i'm mis-reading this.